incubator/hnc/test/e2e/demo_test.go

```diff
@@ -52,7 +55,7 @@ var _ = Describe("Demo", func() {
 		// Now, if we check service-1 again, we’ll see all the rolebindings we expect:
 		RunShouldContainMultiple([]string{"hnc.x-k8s.io/inheritedFrom=acme-org", "hnc.x-k8s.io/inheritedFrom=team-a"},
 			propogationTimeout, "kubectl -n", nsService1, "describe roles")
-		RunShouldContainMultiple([]string{"Role/acme-org-sre", "Role/team-a-sre"}, propogationTimeout, "kubectl -n", nsService1, "get rolebindings")
+		RunShouldContainMultiple([]string{"acme-org-sre", "team-a-sre"}, propogationTimeout, "kubectl -n", nsService1, "get rolebindings")
```

Comment:
Do you have any idea what changed here? `kubectl -n <foo> get rolebindings` isn't an HNC thing that's changed. Also, why do we not see failures in v0.5? Is this test not checked into v0.5?

Reply:
I'm still not sure where this format change come from, but no this test is not checked into v0.5

Reply:
Weird. Maybe it never passed, or it passed with an old version of kubectl? But knowing it's not in v0.5 is probably good enough.